src/frontend/src/lib/utils/tokens.utils.ts

```diff
@@ -40,3 +45,44 @@ export const sortTokens = ({
 			a.name.localeCompare(b.name) ||
 			a.network.name.localeCompare(b.network.name)
 	);
+
+export const pinTokensWithBalanceAtTop = ({
```

Comment:
Can you add a comment to summarize the logic?

Reply:
on it